dev-tools/xtask/src/virtual_hardware.rs

```diff
@@ -34,6 +34,10 @@ enum Commands {
         #[clap(long)]
         gateway_ip: Option<String>,
 
+        /// The configured mode for softnpu
```

Comment:
Should this be a bool? we only know the value `zone`, and all other values are ignored

Reply:
If we change this to a bool, i think the environment variable name would need to change to something like softnpu_zone_mode. The idea behind using a string was to be able to accommodate multiple modes, it just turns out right now that there is nothing special the virtual hardware setup needs to do for "propolis" mode.